utils/lru_test.go

```diff
@@ -20,49 +21,42 @@ func TestLRU(t *testing.T) {
 	for i := 0; i < 256; i++ {
 		l.Add(i, i)
 	}
-	if l.Len() != 128 {
-		t.Fatalf("bad len: %v", l.Len())
-	}
+	require.Equalf(t, 128, l.Len(), "bad len: %v", l.Len())
```

Comment:
Could we use `require.Len` to be more semantic here?

Reply:
Not sure why but this was reverted to `require.Equalf` from `require.Len`

Reply:
I'm changed it to `require.Equalf` from `reuire.Len`, `require.Len` expects a slice. but l isn't slice. 
If you've any suggestion regarding this please assist me.

Reply:
Thanks @a8uhnf! I had completely missed that it was not a slice but a `Cache` struct.